extensions/amp-analytics/0.1/cid-impl.js

```diff
@@ -110,17 +110,14 @@ export class Cid {
    *      This promise may take a long time to resolve if consent isn't
    *      given.
    */
-  get(externalCidScope, consent, opt_persistenceConsent) {
-    /** @type {!GetCidDef} */
-    let getCidStruct;
-    if (typeof externalCidScope == 'string') {
-      getCidStruct = {scope: externalCidScope};
-    } else {
-      getCidStruct = /** @type {!GetCidDef} */ (externalCidScope);
-    }
-    user().assert(/^[a-zA-Z0-9-_.]+$/.test(getCidStruct.scope),
-        'The client id name must only use the characters ' +
-        '[a-zA-Z0-9-_.]+\nInstead found: %s', getCidStruct.scope);
+  get(getCidStruct, consent, opt_persistenceConsent) {
+    dev().assert(typeof getCidStruct != 'string', ''); // DO-NOT-SUBMIT
```

Comment:
yes. added to catch any code that are passing `string` . will remove once all tests pass